### DISCUSSION TITLE
Meeting Agenda 3/6

### DISCUSSION THREAD

**Discussion by dyc3** [dyc3/senior-design#328] [Meeting Agendas/Minutes]
1. Review tasks
    1. Rebase dyc3/opentogethertube#1459
3. Merge dyc3/opentogethertube#1316
4. Turn in weekly report
5. Make decision on vis node layouts
6. Meet with Darian

**Comment by moreno-michael** [comment]
Meeting minutes 3/7

1. Task Reveiew
- Issue #1295 in progress, Issue #1294 todo
- Issues #1409 and #1427 in progress
- Issues #1338 and #1385 in progress, Issue #1339 completed
- Rebase completed

2. Merge completed and everyone successfully upgraded to yarn 4

3. Weekly report turned in

4. Vis node layout discussion
- Have a concrete approach set by end of march
- Linear layout may be best approach
- Merge [ott/#1434](https://github.com/dyc3/opentogethertube/pull/1434) to be able to display multiple interconnected balancers

5. Meet with Darian
- Expo info discussed
- Class schedule discussed
